lib/v2/screens/explore_screens/vote_screens/proposal_details/proposal_details_screen.dart

```diff
@@ -0,0 +1,254 @@
+                                      ),
+                                      const SizedBox(height: 10.0),
+                                      Row(children: [Text('TODO', style: Theme.of(context).textTheme.subtitle2)]),
+                                    ],
+                                  ),
+                                ),
+                              )
+                            ],
+                          ),
+                        ],
+                      ),
+                    ),
+                    const DividerJungle(),
+                    Padding(
+                      padding: const EdgeInsets.all(16.0),
+                      child: Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          Text(
+                            'Recipient: %s '.i18n.fill(["${proposal.recipient}"]),
+                            style: Theme.of(context).textTheme.subtitle3OpacityEmphasis,
+                          ),
+                          const SizedBox(height: 8),
+                          Text(
+                            'Requested: %s SEEDS'.i18n.fill(['${proposal.quantity}']),
+                            style: Theme.of(context).textTheme.subtitle3OpacityEmphasis,
+                          ),
+                          const SizedBox(height: 8),
+                          Text(
+                            'Type: %s '
+                                .i18n
+                                .fill([proposal.campaignType == 'alliance' ? "Alliance".i18n : "Campaign".i18n]),
+                            style: Theme.of(context).textTheme.subtitle3OpacityEmphasis,
+                          ),
+                          const SizedBox(height: 8),
+                          Text(
+                            'Status: %s '.i18n.fill(["${proposal.status}"]),
+                            style: Theme.of(context).textTheme.subtitle3OpacityEmphasis,
+                          ),
+                          const SizedBox(height: 8),
+                          Text(
+                            'Stage: %s '.i18n.fill(["${proposal.stage}"]),
+                            style: Theme.of(context).textTheme.subtitle3OpacityEmphasis,
+                          ),
+                          const SizedBox(height: 8),
+                          RichText(
+                            text: TextSpan(
+                              children: [
+                                TextSpan(
+                                  text: 'URL: '.i18n,
+                                  style: Theme.of(context).textTheme.subtitle3OpacityEmphasis,
+                                ),
+                                TextSpan(
+                                  text: proposal.url,
+                                  style:
+                                      Theme.of(context).textTheme.subtitle3OpacityEmphasis.copyWith(color: Colors.blue),
```

Comment:
I already ask to Jason to provide this link style.

Reply:
Fixed.
![image](https://user-images.githubusercontent.com/42857405/122979064-bad67180-d35c-11eb-9777-293517be28d4.png)